.github/workflows/build.yml

```diff
@@ -254,10 +258,66 @@ jobs:
           corepack prepare yarn@${{ env.YARN_VERSION }} --activate
           yarn set version ${{ env.YARN_VERSION }}
           yarn --version
+      - name: Free up additional disk space
+        run: |
+          ./.github/workflows/scripts/free_gha_space.sh
       - name: Build
         run: |
           make install-prod
-      - name: Upload artifacts
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build notebook images
+        id: build-notebook-images
+        run: |
+          TAG_SUFFIX=$([ -n "${{ github.event.pull_request.number }}" ] && echo "-pr-${{ github.event.pull_request.number }}" || echo "-sha-$(echo "${{ github.sha }}" | cut -c1-8)")
+          echo "Tag suffix: $TAG_SUFFIX"
+
+          WHL_FILE_PATH=$(ls dist/*.whl | head -n 1)
+          WHL_FILE_NAME=$(basename $WHL_FILE_PATH)
+          echo "Wheel file: $WHL_FILE_PATH"
+
+          URLS=""
+          for tag_prefix in $(echo "${{ env.QUAY_WORKBENCH_IMAGES_TAG_PREFIX_CSV }}" | tr ',' '\n'); do
+            LATEST_TAG=$(curl -s "https://quay.io/api/v1/repository/${{ env.QUAY_WORKBENCH_IMAGES_REPOSITORY}}/tag/?onlyActiveTags=true" | \
+              jq -r ".tags | map(select(.name | startswith(\"${tag_prefix}\"))) | .[].name" | \
+              sort -r | head -n 1)
+            echo "Latest tag: $LATEST_TAG"
+
+            REPOSITORY="${{ github.event.pull_request.head.repo.full_name || github.repository }}"
+            echo "Repository: ${REPOSITORY}"
+
+            CONTAINER_IMAGE=ghcr.io/${REPOSITORY}/workbench-images:${LATEST_TAG}${TAG_SUFFIX}
+            echo "Build container image: $CONTAINER_IMAGE"
+
+            docker build -t ${CONTAINER_IMAGE} -f - . <<EOF
+            FROM quay.io/${{ env.QUAY_WORKBENCH_IMAGES_REPOSITORY}}:${LATEST_TAG}
+            COPY $WHL_FILE_PATH /tmp/
+            RUN pip uninstall -y odh-elyra \
+              && pip install /tmp/$WHL_FILE_NAME \
+              && jupyter labextension list
```

Comment:
I would like some visible comment that explains what the image is about. In my understanding it is a latest published quay image from opendatahub-io/notebooks that has the elyra wheel replaced with the latest elyra built from the repo here. It took me a while to find this part here.

Reply:
Please suggest the content of the comment you'd like to see.

Reply:
What I wrote in the comment (assuming it's true). And regarding the location for it, probably it belongs to CONTRIBUTING.md, I guess.